internal/operands/metrics/resources.go

```diff
@@ -7,6 +7,8 @@ import (
 )
 
 const PrometheusRuleName = "prometheus-k8s-rules-cnv"
+const SSPOperatorPartOf = "kubevirt"
```

Comment:
Please rewrite this to:
```
const (
  PrometheusRuleName = "prometheus-k8s-rules-cnv"
  SSPOperatorPartOf = "kubevirt"
  SSPOperatorComponent = "ssp-operator"
)
```
Are these variable  `SSPOperatorPartOf` and `SSPOperatorComponent` will be used in different package(e.g. tests)?
If no, can you please make them non accessible from other packages?